urllib3/poolmanager.py

```diff
@@ -110,8 +116,14 @@ class PoolManager(RequestMethods):
 
     proxy = None
 
-    def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
+    def __init__(self, num_pools=10, headers=None, transport_security_manager=None,
+                 **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+
+        if transport_security_manager is None:
+            transport_security_manager = TransportSecurityManager()
```

Comment:
So I'd like to suggest an alternative: a sentinel object. Generally speaking, if you want to have `None` do something sensible but have a "default" behaviour you'd create a module-scoped variable called something like `_DEFAULT_SENTINEL = object()`, and then set the default value of the kwarg to that. Then you can do identity tests to confirm whether or not the default value was used. I'd be happy to see that used here.